### PR TITLE
Add Privacy Policy message (fixes  #707)

### LINKED ISSUE #707: Privacy Policy
### Describe your matter in detail
As mentioned in #431, the plugin does not collect any data.

It would be really nice to have this statement included in the README.md at the root of the repo.

Since data protection laws are becoming more strict in some european countries, this might help a lot with compliance requirements.

### PR DESCRIPTION
**Pull request by HonoluluHenk**
Fixes #707

### CODE REVIEW

**Comment by Robbendebiene** [comment]
Thanks. I read your issue but I couldn't find the time to answer.

Do you think this is enough or should this perhaps be expanded similar to https://github.com/gorhill/uBlock/wiki/Privacy-policy

**Comment by HonoluluHenk** [comment]
Yes, good idea!
So would you agree with the following text?


Gesturify does not collect any data of any kind.

* Gesturify has no home server.
* Gesturify doesn't embed any analytic or telemetry hooks in its code.
* Gesturify doesn't accept donations or any other form of financing.

**GitHub** is used to host the Gesturify project. **GitHub, Inc**. (a subsidiary of **Microsoft Corporation**) owns **GitHub** and is unrelated to Gesturify.

**Comment by HonoluluHenk** [comment]
As with uBlock: this text could be added to the wiki and the README could link to it.

**Comment by Robbendebiene** [comment]
I would prefer having it in the readme. Also it is called "Gesturefy" not "Gesturify" 😅

> Gesturify doesn't accept donations or any other form of financing.

This is the only point that is not really true, because there is a donation page even though the money goes to charity. I would rather omit this line.

But other than that I'm happy with the text!

File: README.md [hunk 1]
```diff
@@ -49,3 +49,12 @@ See the [releases page](https://github.com/Robbendebiene/Gesturefy/releases) for
...
 This project is licensed under the terms of the [GNU General Public License v3.0](https://github.com/Robbendebiene/Gesturefy/blob/master/LICENSE).
+
+**Privacy:**
+
+Gesturefy does not collect any data of any kind.
+
+* Gesturefy has no home server.
+* Gesturefy doesn't embed any analytic or telemetry hooks in its code.
+
+**GitHub** is used to host the Gesturify project. **GitHub, Inc.** (a subsidiary of **Microsoft Corporation**) owns **GitHub** and is unrelated to Gesturefy.
```
Comment (Robbendebiene) [inline README.md:60]:
```suggestion
**GitHub** is used to host the Gesturefy project. **GitHub, Inc.** (a subsidiary of **Microsoft Corporation**) owns **GitHub** and is unrelated to Gesturefy.
```

**Comment by Robbendebiene** [comment]
Thanks!

**Pull request merged**